server/src/main/java/no/nav/foreldrepenger/vtp/server/api/pdl/PdlLeesahRestTjeneste.java

```diff
@@ -113,9 +147,36 @@ public Response leggTilHendelse(PersonhendelseDto personhendelseDto) {
             personhendelse.set("foedselsdato", fødselsdato.build());
         }
 
+        LOG.info("Publiserer FOEDSELSDATO_V1 på kafka for barn med ident {}, født: {}", barnIdent, fødselshendelseDto.fødselsdato());
         sendHendelsePåKafka(personhendelse.build());
+
+        produserForelderBarnRelasjon(fødselshendelseDto, barnIdent);
+    }
+
+    private void produserForelderBarnRelasjon(FødselshendelseDto fødselshendelseDto, String barnIdent) {
+        List<ForelderBarnRelasjonHendelseDto> forelderBarnRelasjonHendelseDtos = new ArrayList<>();
+        if (fødselshendelseDto.fnrMor() != null) {
+            // Legg til relasjoner for mor
+            leggTilForelderBarnRelasjon(forelderBarnRelasjonHendelseDtos, fødselshendelseDto.endringstype(),
+                    fødselshendelseDto.fnrMor(), barnIdent);
+        } else if (fødselshendelseDto.fnrFar() != null) {
+            // Legg til relasjoner for far
+            leggTilForelderBarnRelasjon(forelderBarnRelasjonHendelseDtos, fødselshendelseDto.endringstype(),
+                    fødselshendelseDto.fnrFar(), barnIdent);
+        }
+
+        forelderBarnRelasjonHendelseDtos.forEach(this::produserForelderBarnRelasjonHendelse);
     }
 
+    private void leggTilForelderBarnRelasjon(List<ForelderBarnRelasjonHendelseDto> dtos,
+                                             String endringstype,
+                                             String forelderFnr,
+                                             String barnFnr) {
+        dtos.add(new ForelderBarnRelasjonHendelseDto(endringstype, forelderFnr, barnFnr, "BARN", "MOR"));
```

Comment:
Denne brukes også for far, i det tilfellet bør far helst få rollen FAR

Reply:
Bra catch 👍